tuf/testutils/repo.go

```diff
@@ -54,16 +52,18 @@ func CreateKey(cs signed.CryptoService, gun data.GUN, role data.RoleName, keyAlg
 }
 
 // CopyKeys copies keys of a particular role to a new cryptoservice, and returns that cryptoservice
-func CopyKeys(t *testing.T, from signed.CryptoService, roles ...data.RoleName) signed.CryptoService {
+func CopyKeys(t *testing.T, from signed.CryptoService, roles ...data.RoleName) (signed.CryptoService, error) {
```

Comment:
Would it make sense to remove the `t *testing.T` requirement from the function signature since it's no longer being used?

Reply:
Yes, looks like that would make sense; I can update